src/Sulu/Bundle/CoreBundle/DataFixtures/replacers.xml

```diff
@@ -1,1214 +0,0 @@
-        <column name="to">W</column>
-    </item>
-    <item>
-        <column name="locale">default</column>
-        <column name="from">ŵ</column>
-        <column name="to">w</column>
-    </item>
-    <item>
-        <column name="locale">default</column>
-        <column name="from">Ŷ</column>
-        <column name="to">Y</column>
-    </item>
-    <item>
-        <column name="locale">default</column>
-        <column name="from">ŷ</column>
-        <column name="to">y</column>
-    </item>
-
-    <!-- special letters -->
-    <item>
-        <column name="locale">default</column>
-        <column name="from">Æ</column>
-        <column name="to">AE</column>
-    </item>
-    <item>
-        <column name="locale">default</column>
-        <column name="from">æ</column>
-        <column name="to">ae</column>
-    </item>
-    <item>
-        <column name="locale">default</column>
-        <column name="from">ß</column>
-        <column name="to">ss</column>
-    </item>
-    <item>
-        <column name="locale">default</column>
-        <column name="from">Œ</column>
-        <column name="to">OE</column>
-    </item>
-    <item>
-        <column name="locale">default</column>
-        <column name="from">œ</column>
-        <column name="to">oe</column>
-    </item>
-    <item>
-        <column name="locale">default</column>
-        <column name="from">Ĳ</column>
-        <column name="to">IJ</column>
-    </item>
-    <item>
-        <column name="locale">default</column>
-        <column name="from">ª</column>
-        <column name="to">a</column>
-    </item>
-    <item>
-        <column name="locale">default</column>
-        <column name="from">º</column>
-        <column name="to">o</column>
-    </item>
-
-    <!-- language specific -->
-    <item>
-        <column name="locale">DE</column>
-        <column name="from">&amp;</column>
-        <column name="to">und</column>
-    </item>
-    <item>
-        <column name="locale">EN</column>
-        <column name="from">&amp;</column>
-        <column name="to">and</column>
-    </item>
-    <item>
-        <column name="locale">FR</column>
-        <column name="from">&amp;</column>
-        <column name="to">et</column>
-    </item>
-    <item>
-        <column name="locale">IT</column>
-        <column name="from">&amp;</column>
-        <column name="to">e</column>
-    </item>
-    <item>
-        <column name="locale">NL</column>
-        <column name="from">&amp;</column>
-        <column name="to">en</column>
-    </item>
-    <item>
-        <column name="locale">ES</column>
-        <column name="from">&amp;</column>
-        <column name="to">y</column>
-    </item>
-    <item>
-        <column name="locale">DE</column>
-        <column name="from"><![CDATA[&]]></column>
-        <column name="to">und</column>
-    </item>
-    <item>
-        <column name="locale">EN</column>
-        <column name="from"><![CDATA[&]]></column>
-        <column name="to">and</column>
```

Comment:
@Warxcell Can you check how the transliterator behaves in this case? An `&` should be replaced by `and` translated to the given language.